src/composables/notifications.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  computed, onBeforeUnmount, onMounted, ref,
+} from '@vue/composition-api';
+import {
+  IAccount, IChainNames,
+  INetwork,
+  INotification,
+  INotificationSetting,
+  NotificationStatus,
+} from '../types';
+import { useGetter, useState } from './vuex';
+import {
+  NOTIFICATION_STATUS_CREATED,
+  NOTIFICATION_STATUS_READ,
+  NOTIFICATION_TYPE_WALLET,
+  executeAndSetInterval,
+  watchUntilTruthy, fetchJson, postJson, NOTIFICATION_ENTITY_TYPE_TIP, AGGREGATOR_URL,
+} from '../popup/utils';
+
+const fetchedNotificationsLimit = 20;
+
+const mountedComponentsCounter = ref<number>(0);
```

Comment:
This counter does not count the mounted components, but components that require polling. So the name is kind of misleading.